apps/storefront/src/pages/QuotesList/index.test.tsx

```diff
@@ -228,11 +309,79 @@ describe('when the user is a B2B customer', () => {
         ),
       );
 
-      const { navigation } = renderWithProviders(<QuotesList />, { preloadedState });
+      const someDraftQuoteItem = buildDraftQuoteItemWith({
+        node: { basePrice: 23, taxPrice: 0, quantity: 2 },
+      });
+      const anotherDraftQuoteItem = buildDraftQuoteItemWith({
+        node: { basePrice: 77, taxPrice: 0, quantity: 2 },
+      });
+
+      const quoteInfo = buildQuoteInfoStateWith({
+        draftQuoteList: [someDraftQuoteItem, anotherDraftQuoteItem],
+      });
+
+      renderWithProviders(<QuotesList />, {
+        preloadedState: { ...preloadedState, quoteInfo },
+      });
+
+      const table = await screen.findByRole('table');
+      const allRows = within(table).getAllByRole('row');
+
+      const rowOfDraftQuote = within(table).getByRole('row', { name: /Draft/ });
+      const rowOfSomeSavedQuote = within(table).getByRole('row', {
+        name: /Some Saved Quote/,
+      });
+
+      // Row 0 is the header row
+      expect(allRows[1]).toBe(rowOfDraftQuote);
+      expect(allRows[2]).toBe(rowOfSomeSavedQuote);
+
+      const allDraftQuoteCells = within(rowOfDraftQuote).getAllByRole('cell');
+
+      // Quote #
+      expect(allDraftQuoteCells[0]).toHaveTextContent('—');
+      // Title
+      expect(allDraftQuoteCells[1]).toHaveTextContent('—');
+      // Sales rep
+      expect(allDraftQuoteCells[2]).toHaveTextContent('—');
+      // Draft quotes are created by the current user in state
+      expect(allDraftQuoteCells[3]).toHaveTextContent('John Doe');
+      // Date created
+      expect(allDraftQuoteCells[4]).toHaveTextContent('—');
+      // Last update
+      expect(allDraftQuoteCells[5]).toHaveTextContent('—');
+      // Expiration date
+      expect(allDraftQuoteCells[6]).toHaveTextContent('—');
+      // Subtotal is the sum of all the items in the draft quote
+      expect(allDraftQuoteCells[7]).toHaveTextContent('$200.00');
```

Comment:
Adding this for reference

<img width="1001" alt="image" src="https://github.com/user-attachments/assets/4d1ea695-36ab-4237-8dff-9340758dde8f" />